src/rastervision/label_stores/object_detection_geojson_file_test.py

```diff
@@ -79,6 +127,33 @@ def tearDown(self):
         self.mock_s3.stop()
         self.temp_dir.cleanup()
 
+    def test_multipolygon_geojson_to_labels(self):
+        geojson = add_classes_to_geojson(self.multipolygon_geojson_dict,
+                                         self.class_map)
+        labels = geojson_to_labels(geojson, self.crs_transformer)
+        label_coordinates = labels.boxlist.get()
+
+        expected_box_coordinates = np.array(
```

Comment:
This only tests that the boxes are correct, but you should also test that the classes and scores in the labels are correct. To make this easier, you should construct an `expected_labels` object and then use the `ObjectDetectionLabels.assert_equal` method.

Reply:
that makes sense, will do